torchvision/ops/poolers.py

```diff
@@ -9,6 +10,158 @@
 from .roi_align import roi_align
 
 
+def _convert_to_roi_format(boxes: List[Tensor]) -> Tensor:
```

Comment:
copy-pasted from  from MultiScaleRoIAlign

Reply:
I understand you just move this outside of the operator to share code, right? If that's the case, I would include this on the refactoring PR.